Composer/packages/client/__tests__/components/expandableNode.test.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
@beyackle I see a bug when i add a skill. Make a change in an LG template of a skill. Do a refresh. We get an unhandled exception

Reply:
This looks fixed, though I could use verification.